Error in build

I get an error when I follow the build instructions.  Here is the output:

```
$ npm run build

> vue-flux@1.0.0 build /path/to/vue-redux/example
> webpack

Container#eachAtRule is deprecated. Use Container#walkAtRules instead.
Container#eachRule is deprecated. Use Container#walkRules instead.
Container#eachDecl is deprecated. Use Container#walkDecls instead.
Hash: 9ddc1cefa99c4358e024
Version: webpack 1.12.9
Time: 3808ms
 Asset    Size  Chunks             Chunk Names
app.js  777 kB       0  [emitted]  app
   [0] multi app 40 bytes {0} [built]
    + 94 hidden modules

ERROR in ../reduxMixinsCreator.js
Module not found: Error: Cannot resolve module 'redux' in /path/to/vue-redux
 @ ../reduxMixinsCreator.js 3:12-28
```

Sure!  I don't remember seeing those warnings the first time around, though:

```
$ npm version
{ npm: '3.3.12',
  ares: '1.10.1-DEV',
  http_parser: '2.6.0',
  icu: '56.1',
  modules: '47',
  node: '5.3.0',
  openssl: '1.0.2e',
  uv: '1.8.0',
  v8: '4.6.85.31',
  zlib: '1.2.8' }
$ git clone https://github.com/yang-wei/vue-redux.git
Cloning into 'vue-redux'...
remote: Counting objects: 235, done.
remote: Total 235 (delta 0), reused 0 (delta 0), pack-reused 235
Receiving objects: 100% (235/235), 30.36 KiB | 0 bytes/s, done.
Resolving deltas: 100% (104/104), done.
Checking connectivity... done.
$ cd vue-redux/example
$ npm install
npm WARN install Couldn't install optional dependency: Unsupported

> expect@1.13.4 postinstall /tmp/vue-redux/example/node_modules/expect
> node ./npm-scripts/postinstall.js

vue-flux@1.0.0 /tmp/vue-redux/example
├─┬ babel@5.8.34 
│ ├─┬ chokidar@1.4.2 
│ │ ├─┬ anymatch@1.3.0 
│ │ │ ├── arrify@1.0.1 
│ │ │ └─┬ micromatch@2.3.7 
│ │ │   ├─┬ arr-diff@2.0.0 
│ │ │   │ └── arr-flatten@1.0.1 
│ │ │   ├── array-unique@0.2.1 
│ │ │   ├─┬ braces@1.8.3 
│ │ │   │ ├─┬ expand-range@1.8.1 
│ │ │   │ │ └─┬ fill-range@2.2.3 
│ │ │   │ │   ├─┬ is-number@2.1.0 
│ │ │   │ │   │ └── kind-of@3.0.2 
│ │ │   │ │   ├── isobject@2.0.0 
│ │ │   │ │   └─┬ randomatic@1.1.5 
│ │ │   │ │     └── kind-of@3.0.2 
│ │ │   │ ├── preserve@0.2.0 
│ │ │   │ └── repeat-element@1.1.2 
│ │ │   ├── expand-brackets@0.1.4 
│ │ │   ├─┬ extglob@0.3.1 
│ │ │   │ ├─┬ ansi-green@0.1.1 
│ │ │   │ │ └── ansi-wrap@0.1.0 
│ │ │   │ └── success-symbol@0.1.0 
│ │ │   ├── filename-regex@2.0.0 
│ │ │   ├─┬ kind-of@3.0.2 
│ │ │   │ └── is-buffer@1.1.0 
│ │ │   ├── normalize-path@2.0.1 
│ │ │   ├─┬ object.omit@2.0.0 
│ │ │   │ ├─┬ for-own@0.1.3 
│ │ │   │ │ └── for-in@0.1.4 
│ │ │   │ └── is-extendable@0.1.1 
│ │ │   ├─┬ parse-glob@3.0.4 
│ │ │   │ ├── glob-base@0.3.0 
│ │ │   │ └── is-dotfile@1.0.2 
│ │ │   └─┬ regex-cache@0.4.2 
│ │ │     ├── is-equal-shallow@0.1.3 
│ │ │     └── is-primitive@2.0.0 
│ │ ├── async-each@0.1.6 
│ │ ├── glob-parent@2.0.0 
│ │ ├── inherits@2.0.1 
│ │ ├─┬ is-binary-path@1.0.1 
│ │ │ └── binary-extensions@1.4.0 
│ │ ├─┬ is-glob@2.0.1 
│ │ │ └── is-extglob@1.0.0 
│ │ └── readdirp@2.0.0 
│ ├─┬ commander@2.9.0 
│ │ └── graceful-readlink@1.0.1 
│ ├── convert-source-map@1.1.2 
│ ├── fs-readdir-recursive@0.1.2 
│ ├─┬ glob@5.0.15 
│ │ ├─┬ inflight@1.0.4 
│ │ │ └── wrappy@1.0.1 
│ │ └── once@1.3.3 
│ ├── lodash@3.10.1 
│ ├─┬ output-file-sync@1.1.1 
│ │ └── xtend@4.0.1 
│ ├── path-exists@1.0.0 
│ ├── path-is-absolute@1.0.0 
│ ├── slash@1.0.0 
│ └── source-map@0.5.3 
├─┬ babel-core@5.8.34 
│ ├── babel-plugin-constant-folding@1.0.1 
│ ├── babel-plugin-dead-code-elimination@1.0.2 
│ ├── babel-plugin-eval@1.0.1 
│ ├── babel-plugin-inline-environment-variables@1.0.1 
│ ├── babel-plugin-jscript@1.0.4 
│ ├── babel-plugin-member-expression-literals@1.0.1 
│ ├── babel-plugin-property-literals@1.0.1 
│ ├── babel-plugin-proto-to-assign@1.0.4 
│ ├── babel-plugin-react-constant-elements@1.0.3 
│ ├── babel-plugin-react-display-name@1.0.3 
│ ├── babel-plugin-remove-console@1.0.1 
│ ├── babel-plugin-remove-debugger@1.0.1 
│ ├── babel-plugin-runtime@1.0.7 
│ ├─┬ babel-plugin-undeclared-variables-check@1.0.2 
│ │ └── leven@1.0.2 
│ ├── babel-plugin-undefined-to-void@1.1.6 
│ ├── babylon@5.8.34 
│ ├── bluebird@2.10.2 
│ ├─┬ chalk@1.1.1 
│ │ ├── ansi-styles@2.1.0 
│ │ ├── escape-string-regexp@1.0.4 
│ │ ├── has-ansi@2.0.0 
│ │ └── supports-color@2.0.0 
│ ├── core-js@1.2.6 
│ ├─┬ debug@2.2.0 
│ │ └── ms@0.7.1 
│ ├─┬ detect-indent@3.0.1 
│ │ ├── get-stdin@4.0.1 
│ │ └── minimist@1.2.0 
│ ├── esutils@2.0.2 
│ ├── globals@6.4.1 
│ ├─┬ home-or-tmp@1.0.0 
│ │ ├── os-tmpdir@1.0.1 
│ │ └── user-home@1.1.1 
│ ├─┬ is-integer@1.0.6 
│ │ └─┬ is-finite@1.0.1 
│ │   └── number-is-nan@1.0.0 
│ ├── js-tokens@1.0.1 
│ ├── json5@0.4.0 
│ ├─┬ line-numbers@0.2.0 
│ │ └── left-pad@0.0.3 
│ ├─┬ minimatch@2.0.10 
│ │ └─┬ brace-expansion@1.1.2 
│ │   ├── balanced-match@0.3.0 
│ │   └── concat-map@0.0.1 
│ ├── private@0.1.6 
│ ├─┬ regenerator@0.8.40 
│ │ ├─┬ commoner@0.10.4 
│ │ │ ├─┬ detective@4.3.1 
│ │ │ │ └── acorn@1.2.2 
│ │ │ ├── iconv-lite@0.4.13 
│ │ │ └── q@1.4.1 
│ │ ├─┬ defs@1.1.1 
│ │ │ ├─┬ alter@0.2.0 
│ │ │ │ └── stable@0.1.5 
│ │ │ ├── ast-traverse@0.1.1 
│ │ │ ├── breakable@1.0.0 
│ │ │ ├── esprima-fb@15001.1001.0-dev-harmony-fb 
│ │ │ ├── simple-fmt@0.1.0 
│ │ │ ├── simple-is@0.2.0 
│ │ │ ├── stringmap@0.2.2 
│ │ │ ├── stringset@0.2.1 
│ │ │ ├── tryor@0.1.2 
│ │ │ └─┬ yargs@3.27.0 
│ │ │   ├─┬ os-locale@1.4.0 
│ │ │   │ └─┬ lcid@1.0.0 
│ │ │   │   └── invert-kv@1.0.0 
│ │ │   ├── window-size@0.1.4 
│ │ │   └── y18n@3.2.0 
│ │ ├── esprima-fb@15001.1001.0-dev-harmony-fb 
│ │ ├─┬ recast@0.10.33 
│ │ │ ├── ast-types@0.8.12 
│ │ │ ├── esprima-fb@15001.1001.0-dev-harmony-fb 
│ │ │ └── source-map@0.5.3 
│ │ └── through@2.3.8 
│ ├─┬ regexpu@1.3.0 
│ │ ├── esprima@2.7.1 
│ │ ├── regenerate@1.2.1 
│ │ ├── regjsgen@0.2.0 
│ │ └─┬ regjsparser@0.1.5 
│ │   └── jsesc@0.5.0 
│ ├── repeating@1.1.3 
│ ├── resolve@1.1.6 
│ ├── shebang-regex@1.0.0 
│ ├── source-map@0.5.3 
│ ├─┬ source-map-support@0.2.10 
│ │ └── source-map@0.1.32 
│ ├── to-fast-properties@1.0.1 
│ ├── trim-right@1.0.1 
│ └── try-resolve@1.0.1 
├─┬ babel-loader@5.4.0 
│ ├─┬ loader-utils@0.2.12 
│ │ └── big.js@3.1.3 
│ └── object-assign@3.0.0 
├── babel-runtime@5.8.34 
├─┬ css-loader@0.18.0 
│ ├─┬ css-selector-tokenizer@0.5.4 
│ │ └── cssesc@0.1.0 
│ ├─┬ cssnano@3.4.0 
│ │ ├─┬ autoprefixer@6.2.3 
│ │ │ ├── browserslist@1.0.1 
│ │ │ ├── caniuse-db@1.0.30000382 
│ │ │ ├── normalize-range@0.1.2 
│ │ │ └── num2fraction@1.2.2 
│ │ ├── decamelize@1.1.2 
│ │ ├── defined@1.0.0 
│ │ ├── indexes-of@1.0.1 
│ │ ├── object-assign@4.0.1 
│ │ ├─┬ postcss-calc@5.0.0 
│ │ │ ├── postcss-message-helpers@2.0.0 
│ │ │ └─┬ reduce-css-calc@1.2.0 
│ │ │   ├── balanced-match@0.1.0 
│ │ │   └─┬ reduce-function-call@1.0.1 
│ │ │     └── balanced-match@0.1.0 
│ │ ├─┬ postcss-colormin@2.1.8 
│ │ │ └─┬ colormin@1.0.6 
│ │ │   ├─┬ color@0.10.1 
│ │ │   │ ├── color-convert@0.5.3 
│ │ │   │ └─┬ color-string@0.3.0 
│ │ │   │   └── color-name@1.1.1 
│ │ │   └── css-color-names@0.0.3 
│ │ ├── postcss-convert-values@2.3.4 
│ │ ├── postcss-discard-comments@2.0.3 
│ │ ├── postcss-discard-duplicates@2.0.0 
│ │ ├── postcss-discard-empty@2.0.0 
│ │ ├─┬ postcss-discard-unused@2.1.0 
│ │ │ └── uniqs@2.0.0 
│ │ ├─┬ postcss-filter-plugins@2.0.0 
│ │ │ └── uniqid@1.0.0 
│ │ ├─┬ postcss-merge-idents@2.1.3 
│ │ │ └── has-own@1.0.0 
│ │ ├── postcss-merge-longhand@2.0.1 
│ │ ├── postcss-merge-rules@2.0.3 
│ │ ├─┬ postcss-minify-font-values@1.0.2 
│ │ │ └── object-assign@4.0.1 
│ │ ├── postcss-minify-gradients@1.0.1 
│ │ ├─┬ postcss-minify-params@1.0.4 
│ │ │ └── alphanum-sort@1.0.2 
│ │ ├── postcss-minify-selectors@2.0.1 
│ │ ├── postcss-normalize-charset@1.1.0 
│ │ ├─┬ postcss-normalize-url@3.0.4 
│ │ │ ├── is-absolute-url@2.0.0 
│ │ │ ├─┬ normalize-url@1.4.0 
│ │ │ │ ├── object-assign@4.0.1 
│ │ │ │ ├── prepend-http@1.0.3 
│ │ │ │ ├─┬ query-string@3.0.0 
│ │ │ │ │ └── strict-uri-encode@1.0.2 
│ │ │ │ └─┬ sort-keys@1.1.1 
│ │ │ │   └── is-plain-obj@1.1.0 
│ │ │ └── object-assign@4.0.1 
│ │ ├── postcss-ordered-values@2.0.2 
│ │ ├── postcss-reduce-idents@2.2.1 
│ │ ├── postcss-reduce-transforms@1.0.3 
│ │ ├─┬ postcss-svgo@2.1.1 
│ │ │ ├── is-svg@1.1.1 
│ │ │ └─┬ svgo@0.6.1 
│ │ │   ├── coa@1.0.1 
│ │ │   ├── colors@1.1.2 
│ │ │   ├─┬ csso@1.4.4 
│ │ │   │ └── clap@1.0.10 
│ │ │   ├─┬ js-yaml@3.4.6 
│ │ │   │ ├─┬ argparse@1.0.3 
│ │ │   │ │ └── sprintf-js@1.0.3 
│ │ │   │ ├── esprima@2.7.1 
│ │ │   │ └── inherit@2.2.2 
│ │ │   ├── sax@1.1.4 
│ │ │   └── whet.extend@0.9.9 
│ │ ├── postcss-unique-selectors@2.0.1 
│ │ ├── postcss-value-parser@3.2.3 
│ │ └── postcss-zindex@2.0.0 
│ ├─┬ postcss@5.0.13 
│ │ ├── js-base64@2.1.9 
│ │ ├── source-map@0.5.3 
│ │ └── supports-color@3.1.2 
│ ├─┬ postcss-modules-extract-imports@0.0.5 
│ │ └─┬ postcss@4.1.16 
│ │   └── source-map@0.4.4 
│ ├─┬ postcss-modules-local-by-default@0.0.12 
│ │ └─┬ postcss@4.1.16 
│ │   └── source-map@0.4.4 
│ ├─┬ postcss-modules-scope@0.0.8 
│ │ └─┬ postcss@4.1.16 
│ │   └── source-map@0.4.4 
│ └── source-list-map@0.1.5 
├─┬ expect@1.13.4 
│ ├─┬ is-equal@1.4.2 
│ │ ├─┬ has@1.0.1 
│ │ │ └── function-bind@1.0.2 
│ │ ├── is-arrow-function@2.0.3 
│ │ ├── is-boolean-object@1.0.0 
│ │ ├── is-callable@1.1.1 
│ │ ├── is-date-object@1.0.1 
│ │ ├── is-generator-function@1.0.3 
│ │ ├── is-number-object@1.0.3 
│ │ ├── is-string@1.0.4 
│ │ ├── is-symbol@1.0.1 
│ │ └─┬ object.entries@1.0.3 
│ │   ├─┬ define-properties@1.1.2 
│ │   │ ├── foreach@2.0.5 
│ │   │ └── object-keys@1.0.9 
│ │   └─┬ es-abstract@1.5.0 
│ │     └── es-to-primitive@1.1.0 
│ ├── is-regex@1.0.3 
│ └── object-inspect@1.1.0 
├─┬ html-loader@0.3.0 
│ ├── fastparse@1.1.1 
│ ├─┬ html-minifier@0.7.2 
│ │ ├─┬ change-case@2.3.1 
│ │ │ ├── camel-case@1.2.2 
│ │ │ ├── constant-case@1.1.2 
│ │ │ ├── dot-case@1.1.2 
│ │ │ ├── is-lower-case@1.1.3 
│ │ │ ├── is-upper-case@1.1.2 
│ │ │ ├── lower-case@1.1.3 
│ │ │ ├── lower-case-first@1.0.2 
│ │ │ ├── param-case@1.1.2 
│ │ │ ├── pascal-case@1.1.2 
│ │ │ ├── path-case@1.1.2 
│ │ │ ├── sentence-case@1.1.3 
│ │ │ ├── snake-case@1.1.2 
│ │ │ ├── swap-case@1.1.2 
│ │ │ ├── title-case@1.1.2 
│ │ │ ├── upper-case@1.1.3 
│ │ │ └── upper-case-first@1.1.2 
│ │ ├─┬ clean-css@3.1.9 
│ │ │ ├── commander@2.6.0 
│ │ │ └── source-map@0.1.43 
│ │ ├─┬ cli@0.6.6 
│ │ │ ├── exit@0.1.2 
│ │ │ └─┬ glob@3.2.11 
│ │ │   └── minimatch@0.3.0 
│ │ ├─┬ concat-stream@1.4.10 
│ │ │ ├── readable-stream@1.1.13 
│ │ │ └── typedarray@0.0.6 
│ │ ├── relateurl@0.2.6 
│ │ └─┬ uglify-js@2.4.24 
│ │   ├── async@0.2.10 
│ │   ├── source-map@0.1.34 
│ │   └─┬ yargs@3.5.4 
│ │     └── window-size@0.1.0 
│ └─┬ source-map@0.1.31 
│   └── amdefine@1.0.0 
├─┬ mocha@2.3.4 
│ ├── commander@2.3.0 
│ ├── diff@1.4.0 
│ ├── escape-string-regexp@1.0.2 
│ ├─┬ glob@3.2.3 
│ │ ├── graceful-fs@2.0.3 
│ │ └─┬ minimatch@0.2.14 
│ │   ├── lru-cache@2.7.3 
│ │   └── sigmund@1.0.1 
│ ├── growl@1.8.1 
│ ├─┬ jade@0.26.3 
│ │ ├── commander@0.6.1 
│ │ └── mkdirp@0.3.0 
│ ├─┬ mkdirp@0.5.0 
│ │ └── minimist@0.0.8 
│ └── supports-color@1.2.0 
├─┬ node-libs-browser@0.5.3 
│ ├── assert@1.3.0 
│ ├─┬ browserify-zlib@0.1.4 
│ │ └── pako@0.2.8 
│ ├─┬ buffer@3.6.0 
│ │ ├── base64-js@0.0.8 
│ │ ├── ieee754@1.1.6 
│ │ └── isarray@1.0.0 
│ ├─┬ console-browserify@1.1.0 
│ │ └── date-now@0.1.4 
│ ├── constants-browserify@0.0.1 
│ ├─┬ crypto-browserify@3.2.8 
│ │ ├── pbkdf2-compat@2.0.1 
│ │ ├── ripemd160@0.2.0 
│ │ └── sha.js@2.2.6 
│ ├── domain-browser@1.1.7 
│ ├── events@1.1.0 
│ ├─┬ http-browserify@1.7.0 
│ │ └── Base64@0.2.1 
│ ├── https-browserify@0.0.0 
│ ├── os-browserify@0.1.2 
│ ├── path-browserify@0.0.0 
│ ├── process@0.11.2 
│ ├── punycode@1.4.0 
│ ├── querystring-es3@0.2.1 
│ ├─┬ readable-stream@1.1.13 
│ │ ├── core-util-is@1.0.2 
│ │ └── isarray@0.0.1 
│ ├─┬ stream-browserify@1.0.0 
│ │ └── readable-stream@1.1.13 
│ ├── string_decoder@0.10.31 
│ ├── timers-browserify@1.4.2 
│ ├── tty-browserify@0.0.0 
│ ├─┬ url@0.10.3 
│ │ ├── punycode@1.3.2 
│ │ └── querystring@0.2.0 
│ ├── util@0.10.3 
│ └─┬ vm-browserify@0.0.4 
│   └── indexof@0.0.1 
├── redux@3.0.5 
├── redux-logger@1.0.9 
├── style-loader@0.12.4 
├─┬ vue@0.12.16 
│ └─┬ envify@3.4.0 
│   └─┬ jstransform@10.1.0 
│     ├── base62@0.1.1 
│     └── esprima-fb@13001.1001.0-dev-harmony-fb 
├── UNMET PEER DEPENDENCY vue-html-loader@^1.0.0
├─┬ vue-loader@3.0.4 
│ ├── hash-sum@1.0.2 
│ ├── parse5@1.5.1 
│ ├─┬ postcss@4.1.16 
│ │ ├── es6-promise@2.3.0 
│ │ └── source-map@0.4.4 
│ ├─┬ postcss-nested@0.3.2 
│ │ └─┬ postcss@4.1.16 
│ │   └── source-map@0.4.4 
│ └─┬ postcss-selector-parser@1.3.0 
│   ├── flatten@0.0.1 
│   └── uniq@1.0.1 
├─┬ webpack@1.12.9 
│ ├── async@1.5.0 
│ ├── clone@1.0.2 
│ ├─┬ enhanced-resolve@0.9.1 
│ │ ├── graceful-fs@4.1.2 
│ │ └── memory-fs@0.2.0 
│ ├── esprima@2.7.1 
│ ├── interpret@0.6.6 
│ ├─┬ memory-fs@0.3.0 
│ │ ├─┬ errno@0.1.4 
│ │ │ └── prr@0.0.0 
│ │ └─┬ readable-stream@2.0.5 
│ │   ├── process-nextick-args@1.0.6 
│ │   └── util-deprecate@1.0.2 
│ ├─┬ mkdirp@0.5.1 
│ │ └── minimist@0.0.8 
│ ├─┬ optimist@0.6.1 
│ │ ├── minimist@0.0.10 
│ │ └── wordwrap@0.0.2 
│ ├─┬ supports-color@3.1.2 
│ │ └── has-flag@1.0.0 
│ ├── tapable@0.1.10 
│ ├─┬ uglify-js@2.6.1 
│ │ ├── async@0.2.10 
│ │ ├── source-map@0.5.3 
│ │ ├── uglify-to-browserify@1.0.2 
│ │ └─┬ yargs@3.10.0 
│ │   ├── camelcase@1.2.1 
│ │   ├─┬ cliui@2.1.0 
│ │   │ ├─┬ center-align@0.1.2 
│ │   │ │ ├─┬ align-text@0.1.3 
│ │   │ │ │ ├── kind-of@2.0.1 
│ │   │ │ │ ├── longest@1.0.1 
│ │   │ │ │ └── repeat-string@1.5.2 
│ │   │ │ └── lazy-cache@0.2.7 
│ │   │ └── right-align@0.1.3 
│ │   └── window-size@0.1.0 
│ ├─┬ watchpack@0.2.9 
│ │ └── async@0.9.2 
│ └─┬ webpack-core@0.6.8 
│   └── source-map@0.4.4 
└─┬ webpack-dev-server@1.14.0 
  ├─┬ compression@1.6.0 
  │ ├─┬ accepts@1.3.0 
  │ │ └── negotiator@0.6.0 
  │ ├── bytes@2.1.0 
  │ ├─┬ compressible@2.0.6 
  │ │ └── mime-db@1.20.0 
  │ ├── on-headers@1.0.1 
  │ └── vary@1.1.0 
  ├── connect-history-api-fallback@1.1.0 
  ├─┬ express@4.13.3 
  │ ├─┬ accepts@1.2.13 
  │ │ └── negotiator@0.5.3 
  │ ├── array-flatten@1.1.1 
  │ ├── content-disposition@0.5.0 
  │ ├── content-type@1.0.1 
  │ ├── cookie@0.1.3 
  │ ├── cookie-signature@1.0.6 
  │ ├── depd@1.0.1 
  │ ├── escape-html@1.0.2 
  │ ├── etag@1.7.0 
  │ ├─┬ finalhandler@0.4.0 
  │ │ └── unpipe@1.0.0 
  │ ├── fresh@0.3.0 
  │ ├── merge-descriptors@1.0.0 
  │ ├── methods@1.1.1 
  │ ├─┬ on-finished@2.3.0 
  │ │ └── ee-first@1.1.1 
  │ ├── parseurl@1.3.0 
  │ ├── path-to-regexp@0.1.7 
  │ ├─┬ proxy-addr@1.0.10 
  │ │ ├── forwarded@0.1.0 
  │ │ └── ipaddr.js@1.0.5 
  │ ├── qs@4.0.0 
  │ ├── range-parser@1.0.3 
  │ ├─┬ send@0.13.0 
  │ │ ├── destroy@1.0.3 
  │ │ └── statuses@1.2.1 
  │ ├── serve-static@1.10.0 
  │ ├─┬ type-is@1.6.10 
  │ │ └── media-typer@0.3.0 
  │ ├── utils-merge@1.0.0 
  │ └── vary@1.0.1 
  ├─┬ http-proxy@1.12.0 
  │ ├── eventemitter3@1.1.1 
  │ └── requires-port@0.0.1 
  ├─┬ serve-index@1.7.2 
  │ ├─┬ accepts@1.2.13 
  │ │ └── negotiator@0.5.3 
  │ ├── batch@0.5.2 
  │ ├── http-errors@1.3.1 
  │ └── mime-types@2.1.8 
  ├─┬ sockjs@0.3.15 
  │ ├─┬ faye-websocket@0.9.4 
  │ │ └─┬ websocket-driver@0.6.3 
  │ │   └── websocket-extensions@0.1.1 
  │ └── node-uuid@1.4.7 
  ├─┬ sockjs-client@1.0.3 
  │ ├─┬ eventsource@0.1.6 
  │ │ └── original@1.0.0 
  │ ├── faye-websocket@0.7.3 
  │ ├── json3@3.3.2 
  │ └─┬ url-parse@1.0.5 
  │   ├── querystringify@0.0.3 
  │   └── requires-port@1.0.0 
  ├── stream-cache@0.0.2 
  ├─┬ strip-ansi@3.0.0 
  │ └── ansi-regex@2.0.0 
  ├── supports-color@3.1.2 
  └─┬ webpack-dev-middleware@1.4.0 
    └── mime@1.3.4 

npm WARN EPEERINVALID vue-loader@3.0.4 requires a peer of vue-html-loader@^1.0.0 but none was installed.
npm WARN EPACKAGEJSON vue-flux@1.0.0 No description
$ npm run build

> vue-flux@1.0.0 build /tmp/vue-redux/example
> webpack

Container#eachAtRule is deprecated. Use Container#walkAtRules instead.
Container#eachRule is deprecated. Use Container#walkRules instead.
Container#eachDecl is deprecated. Use Container#walkDecls instead.
Hash: efa11b6164c92ccbd103
Version: webpack 1.12.9
Time: 2750ms
 Asset    Size  Chunks             Chunk Names
app.js  776 kB       0  [emitted]  app
   [0] multi app 40 bytes {0} [built]
    + 93 hidden modules

ERROR in ../reduxMixinsCreator.js
Module not found: Error: Cannot resolve module 'redux' in /tmp/vue-redux
 @ ../reduxMixinsCreator.js 3:12-28

ERROR in ./src/component/items.vue
Module not found: Error: Cannot resolve module 'vue-html' in /tmp/vue-redux/example/src/component
 @ ./src/component/items.vue 3:26-129
$ find node_modules/ -type d -name redux
node_modules/redux
$ find node_modules/ -type d -name vue-html
$ 
```

I started from scratch again.  I tried installing vue-html-loader manually first, then `npm install` and `npm run build,` and I still get the original error:

```
$ npm run build

> vue-flux@1.0.0 build /tmp/vue-redux/example
> webpack

Container#eachAtRule is deprecated. Use Container#walkAtRules instead.
Container#eachRule is deprecated. Use Container#walkRules instead.
Container#eachDecl is deprecated. Use Container#walkDecls instead.
Hash: 9ed5f16a783236d9de49
Version: webpack 1.12.9
Time: 3397ms
 Asset    Size  Chunks             Chunk Names
app.js  777 kB       0  [emitted]  app
   [0] multi app 40 bytes {0} [built]
    + 94 hidden modules

ERROR in ../reduxMixinsCreator.js
Module not found: Error: Cannot resolve module 'redux' in /tmp/vue-redux
 @ ../reduxMixinsCreator.js 3:12-28
```

The redux module seems to be there, as in previous log excerpt:

```
$ ls node_modules/redux/dist/
redux.js  redux.min.js
```

Thanks for the tutorial reference.  I will definitely have a look.

--- a/example/package.json
+++ b/example/package.json
@@ -19,7 +19,8 @@
   "dependencies": {
     "redux": "^3.0.0",
     "redux-logger": "^1.0.6",
-    "vue": "^0.12.7"
+    "vue": "^0.12.7",
+    "vue-redux": "^0.0.1"
   },
   "devDependencies": {
     "babel": "^5.8.23",

--- a/example/src/app.js
+++ b/example/src/app.js
@@ -10,7 +10,7 @@ const createStoreWithMiddleware = applyMiddleware(logger)(createStore)
 const store = createStoreWithMiddleware(reducers)
 
 import * as ItemAction from './action/ItemAction'
-import reduxMixinsCreator from '../../reduxMixinsCreator'
+import * as reduxMixinsCreator from 'vue-redux'
 const reduxMixins = reduxMixinsCreator(ItemAction)
 
 new Vue({

